src/main/resources/static/app/lab-request/LabRequestController.js

```diff
@@ -47,6 +47,10 @@ angular.module('ProcessApp.controllers')
                 show: false
             });
 
+            $scope.isMaterialsRequest = function(request) {
```

Comment:
Can be simplified to `$scope.isMaterialsRequest = Request.isMaterialsRequest;`
Same fix needed in `RequestController.js` for `upload-file.html`.

Reply:
ah ok, this is updated now